Ensure a logo width of lower than 100 cannot be saved

nickcernis opened the pull request:
**Summary of change:**
Even though the “maximum logo width” field has a min attribute of 100, a value of lower than 100 can still be saved if entered manually rather than using the input field stepper controls.

This commit adds a custom sanitization callback to prevent values under 100 from being saved.

Reported by Marcy Diaz in Slack.

**Have the changes in this PR been added to the documentation for this project?**
Does not apply

**How to test:**
1. Save a logo at Customize -> Site Identity
2. Set the maximum width field to 50.
3. Publish the changes.
4. Confirm that the maximum width field value now reads 100 when refreshing the Customizer.
5. Confirm that the image appears at a maximum width of 100 and not 50.

**Suggested Changelog Entry:**
Ensure logos cannot appear at a width lower than 100px wide.

dreamwhisper commented:
@nickcernis With this, it still appears that the width saves with the lower value to the user. if you close and reopen the customizer, it then shows 100 in the field. Perhaps we should use a `validate_callback` instead. It also provides useful error messages and prevents saving the customizer until it is correct. Thoughts?

```php
	$wp_customize->add_setting(
		'genesis_sample_logo_width',
		[
			'default'           => 350,
			'sanitize_callback' => 'absint',
			'validate_callback' => 'genesis_sample_validate_logo_width',
		]
	);

function genesis_sample_validate_logo_width( $validity, $value ) {

	$value = intval( $value );
	if ( empty( $value ) || ! is_numeric( $value ) ) {
		$validity->add( 'required', __( 'You must supply a valid number.', 'genesis-sample' ) );
	} elseif ( $value < 100 ) {
		$validity->add( 'logo_too_small', __( 'The minimum logo width is 100px.', 'genesis-sample' ) );

	}

	return $validity;

}
```
<img width="302" alt="Screen Shot 2019-08-12 at 12 42 18 PM" src="https://user-images.githubusercontent.com/1271053/62881880-c5eefe00-bcfe-11e9-986a-81da9d8fd23f.png">

nickcernis commented:
@dreamwhisper Nice idea. I like that better too. Switched in https://github.com/studiopress/genesis-sample/pull/288/commits/8b4cdcf62e61dddd797d7bc5ffe1ea17be7301cf.

I changed the message to, “The maximum logo width cannot be less than 100.” (“The minimum logo width is 100” seems confusing to me because people are setting the maximum logo width, and the field description mentions that, so you have to parse it as “the minimum maximum logo width is 100”. 😵But I'm happy to adjust again if needed.)

dreamwhisper commented:
@nickcernis looks good. I think it does read better without “maximum”.

nickcernis commented:
@dreamwhisper Thanks for the feedback. I removed 'maximum' in https://github.com/studiopress/genesis-sample/pull/288/commits/8f96a899587417080be614e9928dbaecaaada385.
